mlflow/keras.py

```diff
@@ -422,7 +422,7 @@ def on_train_end(self, logs=None):
     @gorilla.patch(keras.Model)
     def fit(self, *args, **kwargs):
         if not mlflow.active_run():
-            mlflow.start_run()
+            try_mlflow_log(mlflow.start_run())
```

Comment:
I think we shouldn't actually call the function, just pass a reference to the function:
```suggestion
            try_mlflow_log(mlflow.start_run)
```

Reply:
Whoops! Slipped my mind, will do.